src/storage/storage.cc

```diff
@@ -302,12 +302,12 @@ Status Storage::Open(bool read_only) {
 
   std::vector<rocksdb::ColumnFamilyDescriptor> column_families;
   // Caution: don't change the order of column family, or the handle will be mismatched
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(rocksdb::kDefaultColumnFamilyName, subkey_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kMetadataColumnFamilyName, metadata_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kZSetScoreColumnFamilyName, subkey_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kPubSubColumnFamilyName, pubsub_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kPropagateColumnFamilyName, propagate_opts));
-  column_families.emplace_back(rocksdb::ColumnFamilyDescriptor(kStreamColumnFamilyName, subkey_opts));
+  column_families.emplace_back(rocksdb::kDefaultColumnFamilyName, subkey_opts);
```

Comment:
Inconsistent. Why not barely `kDefaultColumnFamilyName`?

Reply:
It is also `rocksdb::kDefaultColumnFamilyName` in the original code.

Reply:
Oops. Yes this constant comes from another namespace :)